packages/core/src/button/base-button.element.scss

```diff
@@ -74,12 +71,8 @@
   }
 }
 
-:host([contains-anchor]) .private-host {
-  padding: 0;
-}
-
-:host(:active) .private-host {
-  box-shadow: 0 #{$cds-global-space-1} 0 0 var(--box-shadow-color) inset;
+:host(:active) .private-host::after {
+  box-shadow: 0 #{$cds-global-space-2} 0 0 var(--box-shadow-color) inset;
```

Comment:
Does this have any negative effect on the compact buttons? The reason we'd used the smaller spacing previously was because a larger box-shadow effect introduced visual issues.

Reply:
I'll double check, probably need a selector for each it sounds like

Reply:
I see now, reverted change.